src/index.html

```diff
@@ -1,13 +1,367 @@
+                      >
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="category__subtitle">
+                <a class="category__subtitle-link" href="#">Headphones & earbuds</a>
+              </div>
+            </div>
+
+            <div class="category__item">
+              <div class="category__photos">
+                <div class="grid">
+                  <div class="category--scale grid__item grid__item--tablet-1-3 grid__item--desktop-1-8">
+                    <div class="category__photo-container">
+                      <img
+                        src="./images/gallery/speakers-1.png"
+                        alt="speakers photo"
+                        class="category__photo"
+                      >
+                    </div>
+                  </div>
+                  <div class="category--scale grid__item grid__item--tablet-4-6 grid__item--desktop-9-12">
+                    <div class="category__photo-container">
+                      <img
+                        src="./images/gallery/speakers-2.png"
+                        alt="speakers photo"
+                        class="category__photo"
+                      >
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="category__subtitle">
+                <a class="category__subtitle-link" href="#">Speakers</a>
+              </div>
+            </div>
+
+            <div class="category__item">
+              <div class="category__photos">
+                <div class="grid">
+                  <div class="category--scale grid__item grid__item--tablet-1-2 grid__item--desktop-1-4">
+                    <div class="category__photo-container">
+                      <img
+                        src="./images/gallery/audio-sunglasses-1.png"
+                        alt="headphones earbuds photo"
+                        class="category__photo"
+                      >
+                    </div>
+                  </div>
+                  <div class="category--scale grid__item grid__item--tablet-3-6 grid__item--desktop-5-12">
+                    <div class="category__photo-container">
+                      <img
+                        src="./images/gallery/audio-sunglasses-2.png"
+                        alt="Audio sunglassess photo"
+                        class="category__photo"
+                      >
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <div class="category__subtitle">
+                <a class="category__subtitle-link" href="#">Audio sunglasses</a>
+              </div>
+            </div>
+          </div>
+          <div class="grid">
+            <div class="grid__item grid__item-1-2 grid__item--tablet-2-5 grid__item--desktop-5-8">
+              <a class="button" href="#">View all</a>
```

Comment:
I can't hover it, I guess it is problem with styles of photo above

Reply:
I know about this problem, it's because photos has padding bottom , i quess... & i have no idea how to fix it (
![image](https://user-images.githubusercontent.com/117776241/209655994-d28252e0-ec30-46ea-9236-700ea23e5c4b.png)